release/models/qos/openconfig-qos-elements.yang

```diff
@@ -35,7 +35,13 @@ submodule openconfig-qos-elements {
       packets for transmission, including policer and shaper
       functions";
 
-  oc-ext:openconfig-version "0.11.2";
+  oc-ext:openconfig-version "0.11.3";
+
+  revision "2025-05-22" {
+    description
+      "Add two new enums for scheduler-policy priority.";
```

Comment:
Can we specify the two new enums in the description? "Add WRR and DWRR enums ..."